src/crunch/crn_consts.rs

```diff
@@ -0,0 +1,108 @@
+#![allow(non_upper_case_globals)]
+#![allow(dead_code)]
+
+// Moved Consts from structs.
+pub const cCRNHeaderMinSize: u32 = 62;
```

Comment:
It might make sense to convert most of the values here to usize,
considering that most of them are likely used in the context of indexing.